.github/workflows/test.yml

```
@@ -290,6 +290,16 @@ jobs:
     # Windows). 60m leaves generous margin for the slowest shard.
     timeout-minutes: 60
 
+    env:
+      # Where features/seed.py keeps the shared state directory and the
+      # per-scenario copies. It defaults to $HOME, which on a Windows runner is
+      # C: - the OS disk, and the one that filled up and took a shard down with
+      # an unwritable log file. runner.temp is on the same volume as the
+      # workspace, which the runner reported as 147 GB free against C:'s
+      # handful, and is cleaned between jobs. Set for every OS so there is one
+      # path to reason about rather than a Windows special case.
+      PARTCAD_BEHAVE_DIR: ${{ runner.temp }}/partcad-behave
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)